legal-api/src/legal_api/services/search_service.py

```diff
@@ -16,6 +16,7 @@
 # pylint: disable=singleton-comparison ; pylint does not recognize sqlalchemy ==
 from datetime import datetime, timezone
 from http import HTTPStatus
+from flask import current_app
```

Comment:
Is this import still needed?

Reply:
Never mind. I was looking at a standalone commit instead of the whole PR. Resolving.